paper-webserver.tex

```diff
@@ -273,18 +281,21 @@ \section{DISCUSSION}
 Such tools were often script-based, with custom data formats, and only suitable for small collections of data \cite{de2015trends}.
 Recent efforts \cite{goecks2010galaxy,thomsen2016bacterial} have focused on providing a common web interface for these programs, while still returning the same result files.
 However, many subsets of biology now require the analyses of big-data, where inputs are taken from a variety of analysis programs, and involve large-scale data warehousing \cite{schatz2015biological}.
-The ability to integrate data from different source technologies, merge submissions from other labs, and distribute computations over fault-tolerant systems are now required for the types of analyses that need to be performed \cite{schatz2015biological}.
+The ability to integrate data from different source technologies, merge submissions from other labs, and distribute computations over fault-tolerant systems are now required \cite{schatz2015biological}.
 
-One of the key goals in developing Spfy was to accommodate and store a variety of result formats, and then to make the data from these results retrievable and usable as inputs for downstream analyses, such as predictive biomarker discovery. We have shown how a graph database can accommodate the results from a variety of bioinformatics programs, and how Spfy is performant for data retrieval on the results for multiple analyses among over 10,000 genomes, providing results from big-data comparisons in the same efficiency as old analyses on single files.
+One of the key goals in developing Spfy was to accommodate and store a variety of result formats, and then to make the data from these results retrievable and usable as inputs for downstream analyses, such as predictive biomarker discovery. 
+
+% This has not been shown in this paper; it has only been stated. We should probably have a large-scale analysis for an example, with accompanying times etc.
```

Comment:
noted!

Reply:
I'm going to put the description of the large-scale analysis example into the `Functionality` section (and just mention it in the comparison to BAP). Timings will go in the `Results` section along with other timings.